mantidproject/mantid
# Fix qt3 support warnings group 9

This is group-9 of #16128.

This fixes the Qt3Support warnings for group-9.
Straight forward pull-request as only `setMenuText` was required to be changed to `setText` within group-9.

**To Test:**

Check that the text looks correct around Mantid, particularly in the menus & plots 
Code Review
Check MantidPlot builds successfully
Check all builds pass successfully

_Does not need to be in the release notes._





---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

refs #16128
